### ISSUE TITLE
test_no_double_activation on injector fails on travis and locally with Python 2.7.{6,9}

### ISSUE THREAD

**Issue by mvdoc** [duecredit/duecredit#43]
locally (in virtualenv) with Python 2.7.6 I get

```
======================================================================
FAIL: duecredit.tests.test_injections.test_no_double_activation
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/contematto/virtualenv/duecredit/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/contematto/github/duecredit/duecredit/tests/test_injections.py", line 143, in test_no_double_activation
    assert_true(__builtin__.__import__ is duecredited__import__) # we didn't decorate again
AssertionError: False is not true
-------------------- >> begin captured logging << --------------------
duecredit: WARNING: Seems that we are calling duecredit_importer twice. No harm is done but shouldn't happen
duecredit: WARNING: _orig_import is not yet known, so we haven't decorated default importer yet. Nothing TODO
--------------------- >> end captured logging << ---------------------
```

outside virtualenv with python 2.7.9 works fine. however it fails on travis https://travis-ci.org/duecredit/duecredit/jobs/75193538

**Comment by yarikoptic** [comment]
probably related to #40 
are you consistently getting it with 2.7.6?   could I get access to that env?

**Comment by mvdoc** [comment]
how would I share that env? just tar the env?

**Comment by mvdoc** [comment]
Solved by #44
